qa/integration-tests/src/test/java/io/camunda/zeebe/it/processing/MultipleInvalidResourceDeletionTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.it.processing;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.zeebe.client.api.command.ClientStatusException;
+import io.camunda.zeebe.protocol.Protocol;
+import io.camunda.zeebe.qa.util.actuator.PartitionsActuator;
+import io.camunda.zeebe.qa.util.cluster.TestStandaloneBroker;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration.TestZeebe;
+import io.camunda.zeebe.test.util.junit.AutoCloseResources.AutoCloseResource;
+import io.camunda.zeebe.test.util.junit.RegressionTest;
+import io.grpc.Status.Code;
+import org.junit.jupiter.api.BeforeEach;
+
+@ZeebeIntegration
+public class MultipleInvalidResourceDeletionTest {
```

Comment:
❌ I think it is good that we have such regression test, but still I would like to have a more scoped unit tests/stream platform test, since we know now what the actual bug was.

Reply:
It is explained in the PR description:
> It was not possible to reproduce this with a unit test. So an integration test is added.